v3/src/components/map/components/map-point-layer.tsx

```diff
@@ -79,6 +139,18 @@ export const MapPointLayer = function MapPointLayer({mapLayerModel, onSetPixiPoi
     pixiPointsRef.current.resize(layout.contentWidth, layout.contentHeight)
   }
 
+  const refreshConnectingLines = useCallback(async () => {
```

Comment:
```suggestion
  const refreshConnectingLines = useCallback(() => {
```
We don't seem to be `await`ing anything.